toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -377,15 +470,18 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 	return
 }
 
-func initializeRpmDatabase(installRoot string) (err error) {
-	stdout, stderr, err := shell.Execute("rpm", "--root", installRoot, "--initdb")
-	if err != nil {
-		logger.Log.Warnf("Failed to create rpm database: %v", err)
-		logger.Log.Warn(stdout)
-		logger.Log.Warn(stderr)
-		return
+func initializeRpmDatabase(installRoot string, diffDiskBuild bool) (err error) {
+	if !diffDiskBuild {
+		stdout := ""
```

Comment:
styling: for this kind of variable initiazation where you need to setup multiple values to their default values, a `var` block is recommended.

e.g.

```golang
var (
    stdout string
    stderr string
)

```

Reply:
Fixed.

---
In reply to: [573078336](https://github.com/microsoft/CBL-Mariner/pull/611#discussion_r573078336) [](ancestors = 573078336)